jetty/src/main/scala/org/http4s/server/jetty/JettyBuilder.scala

```diff
@@ -400,6 +444,10 @@ object JettyBuilder {
       case SSLClientAuthMode.Required =>
         sslContextFactory.setNeedClientAuth(true)
     }
+
+  /** The default [[org.eclipse.jetty.server.HttpConfiguration]] to use with jetty. */
+  val defaultJettyHttpConfiguration: HttpConfiguration =
```

Comment:
Does this need to be public?  I guess we're already committed to supporting an`HttpConfiguration` on the API, so the additional liability isn't much.  I could go either way here.

Reply:
@rossabaker I made it public so that someone who wants to make a minor adjustment to the configuration can do so and then just use the rest of the "default" http4s configuration. Of course, since our default configuration right now is just the empty constructor, that doesn't really make a lot of sense. 

I am going to make another PR which attempts to align the various configuration parameters which can be meaningfully shared between server backends, e.g. request header size etc. I'll make this private until that PR.

Reply:
@rossabaker updated